code/game/objects/items/ego_weapons/he.dm

```diff
@@ -1346,3 +1346,71 @@
 	attribute_requirements = list(
 							FORTITUDE_ATTRIBUTE = 40
 							)
+
+/obj/item/ego_weapon/roseate
+	name = "roseate desire"
+	desc = "A salacious cat o' nine tails made from pink ribbons. Merely touching it ushers a primal excitement."
+	special = "The damage of this weapon is significantly lowered depending on the user's temperance attribute."
+	icon_state = "roseate_desire"
+	force = 35
```

Comment:
It's passive actively LOWERS it's damage, with the minimum TYPICALLY being a 20% decrease, to 28 force on a slower than normal weapon. It needs MORE force. Make it WAW-like if you want, the fact it has inverse scaling with temperance nerfs the hell out of it.